tests/yam/migration/migration_unattended.pm

```diff
@@ -55,7 +55,7 @@ sub run {
         reconnect_mgmt_console;
     } else {
         power_action('reboot', textmode => 1, keepconsole => 1, first_reboot => 1);
-        assert_screen([qw(grub-menu-migration migration-running)]);
+        assert_screen([qw(grub-menu-migration migration-running)], 60);
```

Comment:
wasn´t already 60 according to logs? but actually should be 30 default? could you please run verification?

Reply:
that is a pity and probably is coming from medium, could you file a ticket to have default timeout in power for qemu (perhaps for hmc is still needed).